units/CorBuildings/LandDefenceOffence/cormexp.lua

```diff
@@ -25,7 +26,7 @@ return {
 		icontype = "building",
 		idleautoheal = 5,
 		idletime = 1800,
-		maxdamage = 3900,
+		maxdamage = 7800, --3900
```

Comment:
We don't need to keep the old values around as comments, if we want to view history, we have source control.

Reply:
Thanks for catching that